spew/common.go

```diff
@@ -325,7 +326,55 @@ func printHexPtr(w io.Writer, p uintptr) {
 // valuesSorter implements sort.Interface to allow a slice of reflect.Value
 // elements to be sorted.
 type valuesSorter struct {
-	values []reflect.Value
+	values  []reflect.Value
+	strings []string
+	cs      *ConfigState
+}
+
+func newValuesSorter(values []reflect.Value, cs *ConfigState) sort.Interface {
```

Comment:
Please add a comment for this function.  You'll notice pretty much every function in the file is commented and I'd like to keep that consistent.